modules/odc-viewing-application-topology.adoc

```diff
@@ -5,29 +5,23 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can switch to *List view* also and see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
 
+The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the graph view.
 
-The status or phase of the Pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
+You can customize the views as required using the following:
 
-After you create an application and an image is deployed, the status is shown as *Pending*. After the application is built, it is displayed as *Running*.
+* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to resize the *Topology* view to show all components.
+* Use the *Display Options* drop-down list to configure the *Topology* view of the various application groupings. The options are available depending on the types of components deployed in the project:
 
-.Application topology
-image::odc_application_topology.png[]
+** *Mode* (*Connectivity* or *Consumption*)
```

Comment:
Please check with the team and expand this. 
Connectivity and Consumption would need to be individual nested bullets like Virtual Machines etc. and add a small description for both please.

Reply:
Updating the information in the commit as suggested.

Reply:
Done